workspaces/liferay-sample-workspace/configs/local/portal-ext.properties

```diff
@@ -8,4 +8,8 @@
 
 feature.flag.LPS-166479=true
 
-feature.flag.LPS-177027=true
\ No newline at end of file
+feature.flag.LPS-177027=true
+
+feature.flag.LPS-172903=true
```

Comment:
Can you add these to `/workspaces/refresh_liferay_workspace.sh` script?

Reply:
Why are some of them there here, then? :thinking: 

Can we move everything to one place or the other? Otherwise there's no SSOT and that will lead to everyone putting them in random places :grimacing: